languages/tgmpa-vi_VN.po

```diff
@@ -0,0 +1,469 @@
+msgid ""
+msgstr ""
+"Plural-Forms: nplurals=1; plural=0;\n"
+"Project-Id-Version: TGM Plugin Activation by BBLand\n"
```

Comment:
Please fix this to be "TGM Plugin Activation 2.6.1" (or whatever version this is for).